sdk/evaluation/azure-ai-evaluation/README.md

```diff
@@ -1,16 +1,28 @@
 # Azure AI Evaluation client library for Python
 
-We are excited to introduce the public preview of the Azure AI Evaluation SDK. 
+Use Azure AI Evaluation SDK to assess the performance of your generative AI applications. Generative AI application generations are quantitatively measured with mathematical based metrics, AI-assisted quality and safety metrics. Metrics are defined as `evaluators`. Built-in or custom evaluators can provide comprehensive insights into the application's capabilities and limitations.
+
+Use Azure AI Evaluation SDK to:
+- Evaluate existing data from generative ai applications
+- Evaluate generative ai applications
+- Evaluate by generating mathematical, AI-assisted quality and safety metrics
+
+Azure AI SDK provides following to evaluate Generative AI Applications:
+- [Evaluators][evaluators] - Generate scores individually or when used together with `evaluate` API.
+- [Evaluate API][evaluate_api] - Python API to evaluate dataset or application using built-in or custom evaluators.
+
+
+
+## Evaluate API
```

Comment:
My understanding is this is a overview section. Do we really need a level 2 heading here?